### DISCUSSION TITLE
Disable checking on certain variables

### DISCUSSION THREAD

**Discussion by lesandrog** [EWSoftware/VSSpellChecker#313] [Q&A] [answered]
How can I disable checking on certain variables. Example: sql = "select ...".
VB.net

**Comment by EWSoftware** [comment, marked as answer]
There's no code fix provider for VB so I assume you're referring to the string in general.  A string is a string so there's no means of classifying it as anything specific.  You can probably update the ignored keywords in the configuration to include the set of SQL keywords as well for VB files.  Things like field names would need to be ignored through the configuration or Ignore Spelling directives in the source file.
